lms/djangoapps/grades/scores.py

```diff
@@ -188,16 +195,19 @@ def _get_score_from_persisted_or_latest_block(persisted_block, block, weight):
     the latest block content.
     """
     raw_earned = 0.0
+    attempted = False
 
     if persisted_block:
         raw_possible = persisted_block.raw_possible
     else:
         raw_possible = block.transformer_data[GradesTransformer].max_score
 
     if raw_possible is None:
-        return (raw_earned, raw_possible) + (None, None)
+        weighted_scores = (None, None)
     else:
-        return (raw_earned, raw_possible) + weighted_score(raw_earned, raw_possible, weight)
+        weighted_scores = weighted_score(raw_earned, raw_possible, weight)
```

Comment:
Since raw_possible is passed in to weighted_scores, does it make sense to push this whole conditional (L205-208) into that function, and eliminate the conditional here?